test/topology_experimental_raft/test_restart_cluster.py

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+"""
+Test clusters can restart fine after all nodes are stopped gracefully
+"""
+
+import logging
+
+import pytest
+from test.pylib.manager_client import ManagerClient
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.asyncio
+async def test_restart_cluster(manager: ManagerClient) -> None:
+    """Test that cluster can restart fine after all nodes are stopped gracefully"""
+    servers = await manager.servers_add(3)
+
+    logger.info(f"Servers {servers}, gracefully stopping servers {[s.server_id for s in servers]} to check if one will go up")
+    for s in servers:
+        await manager.server_stop_gracefully(s.server_id)
+
+    logger.info(f"Starting servers {[s.server_id for s in servers]}")
+    for s in servers:
+        await manager.server_start(s.server_id)
```

Comment:
no need to `wait_for_cql_and_get_hosts`? in theory what you're doing you're adding something which prevents/delays CQL startup so it'd be nice to check that